install_config/aggregate_logging.adoc

```diff
@@ -788,27 +789,20 @@ $ oc delete daemonset logging-fluentd
 $ oc new-app logging-fluentd-template
 ----
 
-[[fluentd-secure-forward]]
-*Configuring Fluentd to send logs to an external log aggregator*
+[[fluentd-external-log-aggregator]]
+*Configuring Fluentd to Send Logs to an External Log Aggregator*
 
-Fluentd can be configured to send a copy of its logs to an external log aggregator
-that can understand messages from Fluentd. This allows the ability to send logs
-to data stores other than Elasticsearch and enables the ability to further process
-log records after the {product-title} Fluentd has processed them.
+You can configure Fluentd to send a copy of its logs to an external log
+aggregator, and not the default Elasticsearch, using the `secure-forward`
+plug-in. From there, you can further process log records after the
+{product-title} Fluentd has processed them.
```

Comment:
"after the {product-title} Fluentd has processed them" feels awkward. Should the word "hosted" go between the product-title macro and Fluentd? Something else perhaps?